record_adb.py

```diff
@@ -0,0 +1,48 @@
+import sys
+import argparse
+import subprocess
+import time
+import os
+import signal
+
+def main(args):
+    recording_name = ''
+    method = args.input
+    recording_name = './sdcard/test.mp4'
+    if args.output is not None:
+            recording_name = './sdcard/' + args.output
+    if method is 'touch' and args.coordinate_x is None or args.coordinate_y is None:
+        print('USE: -i touch --coordinate-x <coordinate> --coordinate-y <coordinate> to use the touch input')
+        sys.exit()
+    elif method is 'intent':
+        record_with_intent(recording_name)
+    else:
+        simulate_input(args.coordinate_x, args.coordinate_y, recording_name)
+        exit()
+
+
+def record_with_intent(recording_name):
+    record_process = subprocess.Popen(['adb', 'shell','screenrecord'] + [recording_name])
+    time.sleep(3)
+    activity_start = subprocess.Popen(['adb', 'shell', 'am', 'start-activity', 'org.mozilla.fenix.nightly/org.mozilla.fenix.HomeActivity', '--ez finishonboarding true'])
+    pull_recording(recording_name)
+
+def simulate_input(x,y, recording_name):
+    record_process = subprocess.Popen(['adb', 'shell','screenrecord'] + [recording_name])
```

Comment:
Do we ever kill this process? If not, it might still be running on your phone 😬 !

Reply:
Fixed that up :)